src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/_format.py

```diff
@@ -139,3 +183,33 @@ def _get_value(item, *args):
         return str(item) if item else ' '
     except (KeyError, TypeError, IndexError):
         return ' '
+
+
+def _get_build_trigger(image_update_trigger, git_commit_trigger):
+    if git_commit_trigger.strip():
+        return 'Git Commit'
+    if image_update_trigger.strip():
+        return 'Image Update'
+    return 'Manual'
+
+
+def _format_datetime(date_string):
+    from dateutil.parser import parse
+    try:
+        return parse(date_string).strftime("%Y-%m-%dT%H:%M:%SZ")
+    except ValueError:
+        logger.debug("Unable to parse date_string '%s'", date_string)
+        return date_string or ' '
```

Comment:
Why a debug message and potentially unexpected behavior rather than a CLIError?

Reply:
We have a few cases that the response doesn't have the datetime yet. For example, if users trigger a build and immediately get the status of it before it finishes, we don't have a finish time here. This is table formatting path and we thought we can just show empty in the column rather than throwing an error.